plugins/install_all.sh

```diff
@@ -59,4 +59,4 @@ done
 # ensure raven core environment dependencies are still met
 echo "Checking raven core dependencies..."
 cd ..
-conda env update -f environment.yml
+#conda env update -f environment.yml
```

Comment:
we should uncomment this and skip installing the plugins for now since the structure of how ravenML depends on them will change soon.